docs/pages/components/ImpactBadge.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: ImpactBadge
+description: The ImpactBadge is an extension of the Badge component is designed to indicate the type of impact
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/ImpactBadge/index.tsx
+---
+
+import { ImpactBadge } from '@deque/cauldron-react';
+
+```js
+import { ImpactBadge } from '@deque/cauldron-react';
+```
+
+## Example
+
+### Critical
+
+```jsx example
+<ImpactBadge type="critical" />
+<ImpactBadge type="critical" size="small" style={{ marginLeft: 'var(--space-half)' }} />
```

Comment:
We don't typically have styles injected in the examples (particularly because people may copy/paste these examples).

My recommendation here would be to split out the sizes separately.